baseapp/baseapp.go

```diff
@@ -385,7 +385,7 @@ func (app *BaseApp) runTx(isCheckTx bool, txBytes []byte, tx sdk.Tx) (result sdk
 	}
 
 	// Match and run route.
-	msgType := msg.Type()
+	msgType := msg.MsgType()
```

Comment:
It was decided in https://github.com/cosmos/cosmos-sdk/issues/519 not to do this :)